tests/golos.publication_tests.cpp

```diff
@@ -209,45 +184,30 @@ BOOST_FIXTURE_TEST_CASE(create_message, golos_publication_tester) try {
     BOOST_CHECK_EQUAL(success(), post.create_msg({N(chucknorris), "permlink", ref_block_num_brucelee_and_chucknorris}));
 
     check_equal_post(post.get_message({N(brucelee), "permlink", ref_block_num_brucelee_and_chucknorris}), _test_msg);
-    check_equal_content(post.get_content({N(brucelee), "permlink", ref_block_num_brucelee_and_chucknorris}), _test_content);
 
     BOOST_TEST_MESSAGE("--- checking that message wasn't closed.");
     produce_blocks(seconds_to_blocks(post.window));
-    auto msg = post.get_message({N(brucelee), "permlink", ref_block_num_brucelee_and_chucknorris});
-    BOOST_CHECK_EQUAL(msg["closed"].as<bool>(), false);
+    auto msg = post.get_message({N(brucelee), "permlink", ref_block_num_brucelee_and_chucknorris});    
+    BOOST_CHECK_EQUAL(msg.is_null(), false);
 
-    BOOST_TEST_MESSAGE("--- checking that message was closed.");
-    produce_block();
-    msg = post.get_message({N(brucelee), "permlink", ref_block_num_brucelee_and_chucknorris});
-    BOOST_CHECK_EQUAL(msg["closed"].as<bool>(), true);
-
-    auto ref_block_num_jackiechan_and_larimer = control->head_block_header().block_num();
-
-    BOOST_CHECK_EQUAL(success(), post.create_msg({N(jackiechan), "permlink1", ref_block_num_jackiechan_and_larimer},
+    auto ref_block_num_jackiechan = control->head_block_header().block_num();
+    BOOST_CHECK_EQUAL(success(), post.create_msg({N(jackiechan), "permlink1", ref_block_num_jackiechan},
                                                  {N(brucelee), "permlink", ref_block_num_brucelee_and_chucknorris}));
 
     msg = post.get_message({N(brucelee), "permlink", ref_block_num_brucelee_and_chucknorris});
     BOOST_CHECK_EQUAL(msg["childcount"].as<uint64_t>(), 1);
 
-    BOOST_CHECK_EQUAL(err.unregistered_user_ + "dan.larimer", post.create_msg({N(dan.larimer), "Hi", ref_block_num_jackiechan_and_larimer}));
-} FC_LOG_AND_RETHROW()
+    BOOST_TEST_MESSAGE("--- checking that message was closed.");
+    produce_block();
+    msg = post.get_message({N(brucelee), "permlink", ref_block_num_brucelee_and_chucknorris});
+    BOOST_CHECK_EQUAL(msg.is_null(), true);
 
-BOOST_FIXTURE_TEST_CASE(update_message, golos_publication_tester) try {
```

Comment:
We need test for update. And in this test it is necessary to check that we can not change the post after closing.